app/mailers/alert_notifier.rb

```diff
@@ -4,7 +4,7 @@ class AlertNotifier < ActionMailer::Base
 
   def alert(theme, alert, applications, comments = [])
     @alert, @applications, @comments = alert, applications, comments
-
+    @analytics_params = {utm_source: "alert", utm_medium: "email", utm_campaign: "trial_subscriber"}
```

Comment:
We use analytics parameters in other links so I'm not sure it's a good idea to call these *the* analytics parameters when they're only used on the trial subscriber links.